scripts/tests/provisioning/test_rauc_provision.expect

```diff
@@ -10,13 +10,15 @@
 #   3+) ipk feed paths (optional)
 log_file -noappend [lindex $argv 1]
 
-set ipk_feeds [lrange $argv 1 end]
+set ipk_feeds [lrange $argv 2 end]
 # The SLIRP networking static ipv4 to use for the VM host (as viewed from the
 # guest). Can be any ip on the 10.0.2.0/24 subnet. 10.0.2.2 is the QEMU
 # default, if none is specified.
 set vm_host_ip "10.0.2.2"
 # Set the login username
-set USER root
+set USER admin
+
+set RE_PROMPT "(admin|root)@.*# $"
```

Comment:
Why do we need to check for either? We're already setting USER to admin and using admin for logging so if we receive root prompt instead of admin, it'd be wrong

Reply:
I'm just checking for both because they are equivalent on NILRT and I think it's a pain to have to go through and swap them around every time we change the login $USER.